packages/angular_devkit/build_optimizer/src/helpers/ast-utils.ts

```diff
@@ -5,9 +5,12 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
+// tslint:disable-next-line: no-implicit-dependencies
+import * as tslib from 'tslib';
 import * as ts from 'typescript';
 
 const pureFunctionComment = '@__PURE__';
+const tslibHelpers = new Set<string>(Object.keys(tslib));
```

Comment:
Will this pull in anything else besides the helpers?

Reply:
There is nothing else defined apart from the helpers. 

if it makes you feel better, when we can do a filter `Object.keys(tslib).filter(x => x.startsWith('__')`

Reply:
Good idea.  Just in case something changes in the future.

Reply:
Could you add a short comment to explaining the significance of `__`?

Reply:
done